Zend/zend_compile.c

```diff
@@ -32,6 +31,7 @@
 #include "zend_language_scanner.h"
 #include "zend_inheritance.h"
 #include "zend_vm.h"
+#include "zend_attributes.h"
```

Comment:
Not sure why these `#include`s were moved originally, but that's probably no longer needed?

I don't care where they are, just doesn't seem related to this patch.